src/Integration.UnitTests/Binding/CSharpVBBindingOperationTests.cs

```diff
@@ -390,64 +449,6 @@ public void ProjectBindingOperation_Commit_FileIsAlreadyReferenced(string existi
             projectMock.Files.Count().Should().Be(1); // existing file updated
         }
 
-        [TestMethod]
-        public void AddAdditional_NonCriticalException_IsCaughtAndWrapped()
```

Comment:
Did you mean to delete these tests? They're still required.

Reply:
Damn, a merge conflict! I'll return them of course